Fonts.css

```diff
@@ -0,0 +1,344 @@
+
+
+h1 {
+  font-family: Arial;
+  font-size: 40px;
+}
+h2 {
+  font-family: Martial;
+  font-size: 24px;
+}
+header {
+  padding: 30px;
```

Comment:
I generally like using rems for padding too. A case could be made for percents, but it'll scale better with rems